### PR TITLE
Creation pending states

### PR DESCRIPTION
**Pull request by phoebemirman**
add failed state to pending market

### CODE REVIEW

File: packages/augur-ui/src/modules/common/labels.tsx [hunk 1]
```diff
@@ -632,20 +633,31 @@ export const InReportingLabel = (props: InReportingLabelProps) => {
...
-    </ReactTooltip>
+interface PendingLabelProps {
+  status?: string;
+}
+
+export const PendingLabel = (props: PendingLabelProps) => (
+  <span className={classNames(Styles.PendingLabel, {[Styles.Failure]: props.status && props.status === TXEventName.Failure})} data-tip data-for={"processing"}>
+    {(!props.status || props.status === TXEventName.Pending) &&
+      <>
+        <span>Processing <ClipLoader size={8} color="#ffffff" /></span>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/labels.tsx:644]: can style be passed into ClipLoader rather than hardcoded color?

Comment (phoebemirman) [reply packages/augur-ui/src/modules/common/labels.tsx:644]: the ClipLoader is a react package so dnt think so

Comment (bthaile) [reply packages/augur-ui/src/modules/common/labels.tsx:644]: ok, lets go with it.

**Review by bthaile — APPROVED** [review]
looks good

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎊
 To view it, be sure to [join Augur Project](https://dashboard.featurepeek.com/login) on FeaturePeek.

 This link will always point to this PR's latest changes: https://peek.run/654h519

**Pull request merged**